WordPress/Classes/Extensions/WPNoResultsView+MediaPicker.swift

```diff
@@ -0,0 +1,44 @@
+//
```

Comment:
We normally don't use file header comments in your files, because they quickly became outdated.